Documentation/botr/readytorun-composite-format-design.md

```diff
@@ -0,0 +1,94 @@
+ReadyToRun File Format - Composite Format Design Notes
```

Comment:
This should better live under https://github.com/dotnet/coreclr/tree/master/Documentation/design-docs

Reply:
Thanks for pointing that out, moved in 8th commit.